packages/rspack/tests/Stats.test.ts

```diff
@@ -177,4 +177,58 @@ describe("Stats", () => {
 		"
 	`);
 	});
+
+	it("should have reasons", async () => {
+		const stats = await compile({
+			context: __dirname,
+			entry: "./fixtures/main5.js",
+			stats: { reasons: true }
+		});
+		expect(stats?.toJson({}, true).modules).toEqual([
```

Comment:
```suggestion
		expect(stats?.toJson({ all: false, reasons: true }).modules).toMatchInlineSnapshot([
```
Prefer `toMatchInlineSnapshot` instead of `toEqual`, `identifier: "javascript/dynamic|/Users/faga/f/rspack/packages/rspack/tests/fixtures/a.js",` will cause CI failed.

Reply:
thx, I use `toMatchInlineSnapshot` now, but it seems has indent problem(`jest --updateSnapshot` result), does it matter?

Reply:
Doesn't matter if it only update this test, I will review the snapshot.